testdata/fuzz/FuzzReverse/0d9771acdce6836f4eef42317c913a9202e76e81c7feb57be4f6da4d27eb0782

```diff
@@ -0,0 +1,2 @@
+go test fuzz v1
+string("\xce")
```

Comment:
What are these `testdata/fuzz/FuzzReverse/*` files? Failure cases found by the fuzz tests?